README.md

```diff
@@ -126,6 +126,110 @@ const Moltin = MoltinGateway({
 })
 
 ``` 
+### TypeScript Support
+The Moltin SDK is fully supported in Typescript. Each service is wrapped in a namespace which exposes all the necessary interfaces and endpoints that can be consumed.
+
+Each response will likely be returned as either a `Response` (single object returned) or a `ResponsePage` (an array of items) - see `src/types/core.d.ts` for more info.
```

Comment:
```suggestion
Each response will be returned as either a `Response` (single object returned) or a `ResponsePage` (an array of items) - see `src/types/core.d.ts` for more info.
```

Reply:
@shaunmaharaj Only reason I put likely was because of things like auth tokens and stuff, which are not returned. Might change it to, `the majority of responses will be returned`